runtime/src/main/java/org/corfudb/runtime/collections/StreamManager.java

```diff
@@ -88,10 +89,10 @@ public void shutdown() {
     /**
      * Subscribe to updates.
      *
-     * @param streamListener   Client listener.
-     * @param namespace        Namespace of interest.
-     * @param tablesOfInterest Only updates from these tables will be returned.
-     * @param startAddress     Address to start the notifications from.
+     * @param streamListener   client listener
+     * @param namespace        namespace of interest
+     * @param tablesOfInterest only updates from these tables will be returned
+     * @param startAddress     address to start the notifications from
```

Comment:
this method can be made default instead of public

Reply:
Done.